__tests__/endpoints.test.js

```diff
@@ -119,4 +118,57 @@ describe('GET/api/articles/:article_id', ()=>{
         })
     })
 });
+describe('GET/api/articles/3/comments', ()=>{
+    test('sends 200 status code and an array of comments for the selcted article', ()=>{
+        return request(app)
+        .get('/api/articles/3/comments')
+        .expect(200)
+        .then(({body})=>{
+            const {comments} = body;
+            expect(comments.length).toBe(2)
+            const sampleComment = {
+                comment_id: 11,
+                body: "Ambidextrous marsupial",
+                votes: 0,
+                author: "icellusedkars",
+                article_id: 3,
+                created_at: "2020-09-19T23:10:00.000Z",
+              }
+            expect(comments[0]).toMatchObject(sampleComment)
+        })
```

Comment:
I think you should probably check that all the comments match the object -> so you need to have a look at a way of dynamically checking them all :) (a length check and then a forEach is probably the nicest way of doing it!)